travis/build-prereqs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+# This script should build all the OPM dependencies which are installed from source.
+
+function build_dune {
+   project=$1
+   pushd . > /dev/null
+   git clone https://github.com/dune-project/${project}.git
+   cd ${project}
+   git checkout tags/v2.3.1
+   mkdir build
+   cd build
+   cmake ../
+   make
+   popd > /dev/null
+}
+
+
+function build_superlu {
+   pushd . > /dev/null
+   git clone https://github.com/starseeker/SuperLU.git
```

Comment:
> I have just copied the existing travis build setup for SuperLU:

Which is completely reasonable as things go.  That said, continuing to use the Starseeker repository is probably not wise for long-term stability.  I don't know offhand how difficult it would be to base the SuperLU build on an official release tarball from the SuperLU homepage.